app.go

```diff
@@ -179,7 +179,13 @@ func (s *StoreState) SetFeature(name string, val bool) {
 // If an error occurs while trying to update the config the change is rolled back, if the rollback
 // itself fails this function will panic.
 func (s *StoreState) ChangeConfigSetting(name, value string) error {
-	cfg := s.Config()
+	chainCfgVersion1_4 := s.FeatureEnabled(ChainCfgVersion1_4, false)
```

Comment:
I haven't enabled it anywhere yet, don't need a separate feature flag for this fix